plugins/inputs/prometheus/start.go

```diff
@@ -282,6 +284,7 @@ func Start(configFilePath string, receiver storage.Appendable, shutDownChan chan
 const (
 	savedScrapeJobLabel      = "cwagent_saved_scrape_job"
 	savedScrapeInstanceLabel = "cwagent_saved_scrape_instance"
+	scrapeInstanceLabel      = "__address__"
```

Comment:
For some reason prometheus used this for caching the name instead of savedScrapeNameLabel. Once adding this worked

Reply:
Yes, this plugin is still being used. The otel bump forced us to upgrade our Prometheus version in CWA. Causing the issue.